code/game/objects/items/cigs_lighters.dm

```diff
@@ -286,6 +286,8 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	if(glass.reagents.trans_to(src, chem_volume, transferred_by = user)) //if reagents were transferred, show the message
 		to_chat(user, span_notice("You dip \the [src] into \the [glass]."))
 	//if not, either the beaker was empty, or the cigarette was full
+	if(istype(glass, /obj/item/reagent_containers/cup/mortar))
+		return NONE
```

Comment:
Are you sure this is the change you want?
I think this should probably be combined with the lines above, the check for "not being a reagent container"

The interaction _I think_ you have created is that you will "dip" the cigarette in whatever reagents are in the mortar and _then_ insert the cigarette into the mortar, but the user won't get any feedback that they did this.

Reply:
if its above, then you cant dip the cigarette in a full mortar, which i didnt like, and if its half full, the cigarette gets covered in the mortar juice when you dip/insert it. say you inserted a cig in a mortar full of plasma, then removed it. It should be covered in plasma, no? This covers all of that

Reply:
fixed